packages/opentelemetry-instrumentation-xml-http-request/test/xhr.test.ts

```diff
@@ -776,7 +776,8 @@ describe('xhr', () => {
             const span: tracing.ReadableSpan = exportSpy.args[0][0][0];
             const attributes = span.attributes;
             const keys = Object.keys(attributes);
-
+            console.log('KEYS: ', keys);
```

Comment:
I expect the console.log statemenst were just for debugging?

Reply:
🤦 yes. Thanks @Flarna.

Reply:
should we catch it with lint ?

Reply:
yes we absolutely should

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/1861

Reply:
Unrelatedly, the docs-test is failing on this PR now. It appears to be the same story on the master branch.